defaults/main.yml

```diff
@@ -49,3 +49,8 @@ cache_timeout: 600
 keepalived_instances: []
 keepalived_sync_groups: {}
 keepalived_bind_on_non_local: False
+
+# Any strings provided here will appear in the `global_defs` secction of the
+# keepalived configuration file.
+keepalived_global_defs:
+  - enable_script_security
```

Comment:
if we have global_defs as a list, the
```
keepalived_global_defs.items()
```
doesn't make sense. (See comment below)

Reply:
I don't think we should have it by default. Instead we should pass it to the role if someone wants to use it. It also doesn't break upgrades if make it optional.